proprietary/Components/src/denhaag/link-group.tokens.json

```diff
@@ -0,0 +1,20 @@
+{
+  "denhaag": {
+    "link-group": {
+      "image": {
+        "padding-block-end": { "value": "1rem" }
+      },
+      "caption": {
+        "color": { "value": "{denhaag.color.grey.5}" },
+        "font-weight": { "value": "500" }
+      },
+      "navigation-link": {
+        "line-height": { "value": "1.6875rem" }
```

Comment:
If the link is 18px, and line height is 27px, we should use `"value": "1.5"` in my opinion.